lex/util/decoder.go

```diff
@@ -76,6 +76,10 @@ func CborDecodeValue(b []byte) (CBOR, error) {
 
 	t, ok := lexTypesMap[tstr]
 	if !ok {
+		// It is possible to gracefully handle this case, like in JsonDecodeValue,
```

Comment:
`ScanForLinks` should properly advance the reader to the next object, ill mess around with this and see if we can get it working correctly

Reply:
FWIW, we don't necessarily have to do both in one go. Skipping CBOR for now means that firehose consumers will error out on unrecognized types. But having JSON calls not fail alone would be already an improvement.

My largest concern however is that this will trigger new code paths, shifting errors down the road. And it will uncover any bugs that are currently hidden behind a failure to unmarshal the whole message.